tox.ini

```diff
@@ -5,6 +5,7 @@ envlist = py38,py39,py310,py311,py312,mypy,precom
 filterwarnings =
     error
     default::DeprecationWarning
+    ignore:Passing msg:DeprecationWarning
```

Comment:
this deprecation happen in Niquests low-level replacement of urllib3. Nothing to be concerned about.